build/Linux+BSD/portable/x86_64/Recipe

```diff
@@ -42,6 +42,7 @@ apt-get -y install \
   alsa \
   libasound2-dev \
   portaudio19-dev \
+  libportmidi-dev \
```

Comment:
I rebased with lasconic's latest commit to fix the Nightly AppImage building with ubuntu 12.04, and I've updated this x86-64 recipe to use ubuntu's "libportmidi-dev" pacakge.